utils/manage_events.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing, please email
+# opensource@seagate.com or cortx-questions@seagate.com.
+
+
+import logging
+from typing import List
+
+
+from cortx.utils.message_bus import MessageBus, MessageConsumer
+from ha.core.event_manager.event_manager import EventManager
+
+
+class ManageEvents():
+    """
+    Typical use case of this class is as follows
+    subscribe(event list) - subscribe to events for listening, returns topic
+                            which can be used for listening
+    _listen(topic) - reading messages received on above topic
+    unsubscribe(event list) - unsubscribe as we dont want to receive these
+                              events anymore
+    get_message_type() - returns topic just in case you are not maintaining
+                         output of subscribe
+    """
+
+    event_manager = None
+
+    def __init__(self):
+        print('Inside __init__')
+        self.event_manager = EventManager.get_instance()
+
+    def subscribe(self, event_list: List[str]) -> str:
+        logging.info('Subscribing for events: (%s)', str(event_list)[1:-1])
+        kafka_topic_name = self.event_manager.subscribe('hare', event_list)
+        return kafka_topic_name
+
+    # Gets message type
+    def get_message_type(self) -> str:
+        print('get_message_type')
+        return self.event_manager.message_type('hare')
+
+    # Get list of events subscribed by Hare
+    def list(self) -> List[str]:
```

Comment:
* list is not the best name for the method (we can refrain `get_events` - it is clearer from the reader's perspective)
* What does it return? In other words, what are there in these strings? Can we parse those raw values into something meaningful?
* What is the semantics of this call?
   * I understand that this fetches some events from Kafka and return to the client. But what happens to the queue then? Are those messages marked as 'read' by the result of this code? If not, what the user of this class needs to do to make sure that the events are not read twice?

Update: after looking down into `_listen()` method, I stopped to understand what this `list()` method is intended to do :-) perhaps some explanation (like vocabulary - what is the type, what is the event, what do we listen for etc) is needed.

Reply:
Changes method name to 'get_events'
Will add some more info about this method/class